### ISSUE TITLE
Latest version cannot be used with CRUD 3.3

### ISSUE THREAD

**Issue by paneidos** [Laravel-Backpack/Settings#60]
# Bug report

### What I did:
Follow the steps to upgrade to CRUD 3.3

### What I expected to happen:
Latest version of this package to install (2.0.24)

### What happened:
Some old version was installed (2.0.8)

### What I've already tried to fix it:
The dependency in composer.json is specified as "3.2.*", it should be "3.2.*|3.3.*" or something like "^3.2.0". It might solve some other issues, like #58 and #59.

### Backpack, Laravel, PHP, DB version:
0.8, 5.5, 7.0, MariaDB 10.1

**Comment by LTiollier-WebID** [comment]
Hello, I have exactly the same bug.. +1

**Comment by jobith93** [comment]
Hi, same bug please help, with the solution

**Comment by bahman2216** [comment]
Also my problem

**Comment by seandowney** [comment]
Same here, does the MR #62 fix this?

**Comment by paneidos** [comment]
Yes, that PR solves it

**Comment by tabacitu** [comment]
Hi guys,

The PR has been merged on Nov 22 - just tested it again and it fixes the problem. Thanks a lot for the PR, again.

However, as of today Backpack\Demo was NOT using that new version of Backpack\Settings. If you were having this problem with the demo, that was the problem, I've now pushed an update to the demo for that.

In case you're having this problem in your own projects, a ```composer update``` will fix it for you guys.

Cheers!
